First tutorial post and a link on the Docs page.

gracecodes144 opened the pull request:
Trying to start a stream of articles on "day-to-day" use of Harvest (as opposed to technical docs), for people who don't really care what the guts look like and just want to get going with using it on their data. Tried to walk the line between giving an overwhelming amount of information and being useless, so linked to a lot of external docs. The plan is to write future posts that give detailed walkthroughs of the steps outlined in this post, plus start to tackle some of the "extras" (which are arguably essentials).

Feel free to comment on everything from tone to content to grammar.

Commit:
First tutorial post and a link on the Docs page.

bruth commented on _posts/2013-11-12-using-your-data.md:
```diff
@@ -0,0 +1,38 @@
+
+Database connections are handled by the Django framework and defined in the settings files at `myproject/conf`. Check out the [Django tutorial](https://docs.djangoproject.com/en/1.5/intro/tutorial01/#database-setup) on this topic for help, and look for detailed examples in a future post.
+
+Once you connect to a new database (other than the SQLite database that comes with a new harvest project), you will have to rebuild the database tables used by Django and Harvest. Use `./bin/manage.py syncdb` and `./bin/manage.py migrate avocado` in order to do this. Read about these commands at the [Django docs](https://docs.djangoproject.com/en/1.5/ref/django-admin/#syncdb) and the [South docs] ...
```
You can replace the two commands with just `python bin/manage.py syncdb --migrate` which also ensures other apps that uses South migrations will be migrated.

gracecodes144 replied:
I thought about this, and almost did it, but the counterargument is that separating the two commands gives an opportunity to link to both the Django and the South docs, and the clarity of which applications are handling each process. But maybe that's better saved for the in depth posts to follow.

After writing to myself for a second there, I agree to the suggested change.

bruth replied:
> the counterargument is that separating the two commands gives an opportunity to link to both the Django and the South docs

I used to think more this way (I generally like being very explicit), but have changed my view on how much of the underlying implementation should be exposed to end users. It's hard to learn something new and as complex as Harvest and introducing additional terms that require the user to lookup in order to understand what they are only detracts from the flow and purpose of the article.

Commit:
Using your data post edit 1.

bruth commented on _posts/2013-11-12-using-your-data.md:
```diff
@@ -0,0 +1,38 @@
+
+## Create and Publish DataConcepts
+
+Now Harvest is connected to your data, but you need to define the way your data will appear in Harvest before you can start using it. You can create DataConcepts either at the command line or in the Django admin view (which you can view at the `http://localhost:8000/admin/` url if you are using the Django development server). In order for a DataField to show up in the Harvest query view, it must  ...
```
I think mentioning that a quick and dirty way to view fields and their corresponding concepts would be to do `python bin/manage.py avocado init myproject` (this will at least produce something in the UI they can see.. though this behavior is coming in the next release 2.2). The caveat being that you will cover the details of this in a follow-up post.

gracecodes144 replied:
I agree, I just forgot to write that sentence...

On a style note, do you think its better to write example commands as `python bin/manage.py ...` as opposed to `./bin/manage.py ...` as I've been doing?

bruth replied:
I only do that because `manage.py` may not be executable (it should be if generated by the harvest command).

bruth commented on _posts/2013-11-12-using-your-data.md:
```diff
@@ -0,0 +1,38 @@
+
+## Start Adding Extras
+
+Other topics to explore include using more than one DataField in a single DataConcept, using QueryViews and Formatters to change the way DataConcepts appear in Harvest, implementing and recording data model changes using South, saving and sharing queries in Harvest, and using Translators and Exporters to get data out of Harvest. 
```
QueryViews are deprecated because they were never actually used.

Commit:
Using your data edit 2 and date change.

gracecodes144 commented:
Added a sentence about avocado init.

Kept the `./bin/manage.py ...` style because that's how I really work (if people want to make manage.py non-executable then they can figure out that they'll need manually call python).

Removed references to QueryView.

bruth commented:
I wasn't implying that user will make the `manage.py` script non-executable, I am saying that it may not be if they download the harvest template manually or by some other means. I don't believe the manage.py file that a `django-admin.py startproject` is executable by default. Leaving it is fine until someone complains.

gracecodes144 commented:
Hmmm, interesting. I checked out the Django book and it seems their standard is to give examples in the `python manage.py ...` format. So, in the interest of consistency with the rest of the Django world, I'll make the changes.

Commit:
Using your data edit 3 and fixing docs link to post.

murphyke commented:
you're --> your in a couple places
ETL could be a link to http://en.wikipedia.org/wiki/Extract,_transform,_load
